substrate/frame/election-provider-multi-phase/src/signed.rs

```diff
@@ -509,6 +509,11 @@ impl<T: Config> Pallet<T> {
 		let positive_imbalance =
 			T::Currency::deposit_creating(who, reward.saturating_add(call_fee));
 		T::RewardHandler::on_unbalanced(positive_imbalance);
+
+		// register submitter as whitelisted for the next round(s).
+		SignedWhitelist::<T>::mutate(|w| {
+			w.force_push(who.clone());
```

Comment:
hmm, why? I think whitelisted accounts should be determined by the governance, no?

Reply:
The idea of this PR is to whitelist the accounts that have recently (i.e. submitted the last accepted signed solutions). So this works as a sort of temporary reputation system, where recent submitters have the part of the deposit waived.

The main goal is to increase considerably the geometric progression deposit without affecting the set of submitters that usually cooperate with good solutions.

We could also have this whitelist set by governance, but I think that's an overkill for this use case and very much static.